python/friesian/src/bigdl/friesian/feature/table.py

```diff
@@ -387,7 +438,11 @@ def show(self, n=20, truncate=True):
         """
         self.df.show(n, truncate)
 
-    def get_stats(self, columns, aggr):
+    def get_stats(
+        self,
+        columns: Optional[Union[List[str], str]],
```

Comment:
why is this optional?

Reply:
> why is this optional?
It could be None i think?
```
        if columns is None:
            columns = [column for column in self.columns if check_column_numeric(self.df, column)]
```